.circleci/config.yml

```diff
@@ -48,30 +48,24 @@ jobs:
       - image: circleci/python:3.6
```

Comment:
changed this file but it is actually not being used right now...

Reply:
Yeah, it's on my to-do list to switch to CircleCI. (It is way faster)